gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/AccountEndpointHandler.java

```diff
@@ -57,6 +63,8 @@
 public class AccountEndpointHandler {
     private static final Logger LOGGER = LoggerFactory.getLogger(AccountEndpointHandler.class);
     private static final String PASSWORD_KEY = "password";
+    private static final String OLD_PASSWORD_KEY = "oldPassword";
+    public static final String ERROR_MSG_ACCESS_TOKEN_TOO_OLD = "AccessToken too old";
```

Comment:
Suggestion: "Access token does not conform with expiration period. Please generate a new token."